src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -1032,7 +1032,7 @@
   "block.structurize.gray_clay_oak_shingle": "Gray Clay Oak Shingle",
   "block.structurize.down_gated_spruce_paper_timber_frame": "Down Gate Framed Spruce Paper Timber Frame",
   "block.structurize.horizontal_plain_oak_stone_timber_frame": "Plain Horizontal Oak Stone Timber Frame",
-  "item.structurize.caliper": "Caliper",
+  "item.structurize.caliper": "Calipers",
```

Comment:
I would say this is an informal version of this world

Reply:
Should I change it back?

Reply:
I changed it to Calipers because I figured it was like scissors, glasses, etc

Reply:
Well up to you then